cpp/time_.h

```diff
@@ -9,21 +9,54 @@
 
 namespace time_ {
 
-inline void* tzset() {
-  NotImplemented();
+inline void tzset() {
+  ::tzset();
 }
 
-inline int time() {
+inline time_t time() {
   return ::time(nullptr);
 }
 
-// TODO: Should these be bigger integers?
-inline int localtime(int ts) {
-  NotImplemented();
+// NOTE(Jesse): time_t is specified to be an arithmetic type by C++. On most
+// systems it's a 64-bit integer.  64 bits is used because 32 will overflow in
+// 2038.  Someone on a comittee somewhere thought of that when moving to 64-bit
+// architectures to prevent breaking ABI again; on 32-bit systems it's usually
+// 32 bits.  Point being, using anything but the time_t typedef here could
+// (unlikely, but possible) produce weird behavior.
```

Comment:
Hmm then it seems like there will be a bug when the translated code consumes this ...